service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -1329,9 +1331,8 @@ private PaymentRequestDto formPayment(Long orderId, int sumToPay) {
         int lastNumber = order.getPayment().size() - 1;
         PaymentRequestDto paymentRequestDto = PaymentRequestDto.builder()
             .merchantId(Integer.parseInt(merchantId))
-            .orderId(
-                orderId + "_" + order.getCounterOrderPaymentId().toString() + "_"
-                    + order.getPayment().get(lastNumber).getId())
+            .orderId(String.format("%s_%s_%s", orderId, order.getCounterOrderPaymentId(),
```

Comment:
I think we agreed to have some Util class that will incapsulate "%s_%s_%s" logic.